### PR TITLE
fix(doc) Update upgradetoCentreon18.10.rst

### PR DESCRIPTION
**Pull request by Pilow974**
Pour contourner le probleme d'écrasement innoDB entre MariaDB 5.x et MariaDB 10.x dans le cadre de la migration CES 3.4.1/centreon-web 2.8.26 vers Centreon 18.10

### CODE REVIEW

File: doc/fr/migration/upgradetoCentreon18.10.rst [hunk 1]
```diff
@@ -67,6 +67,41 @@ suivants : ::
...
+        # drop database centreon_storage;
+        # create database centreon;
+        # create database centreon_storage;
+        
+    #. Editer les fichiers de dump (vi, nano, ...) dans /tmp/ et rajouter
+        # use centreon; en haut de centreon.sql
+        # use centreon_storage; en haut de centreon_storage.sql
+    
+    #. Importer les dumps
+        # mysql -u root -p </tmp/centreon.sql
```
Comment (lpinsivy) [inline doc/fr/migration/upgradetoCentreon18.10.rst]: Should be more simple to remote edition of SQL dump dans execute directly:

```
# mysql -u root centreon -p </tmp/centreon.sql
# mysql -u root centreon_storage -p </tmp/centreon_storage.sql
```

**Comment by lpinsivy** [comment]
@Pilow974 it is possible to have the same translated in EN?

File: doc/fr/migration/upgradetoCentreon18.10.rst (hunk 1 above)
Comment (Pilow974) [reply doc/fr/migration/upgradetoCentreon18.10.rst]: indeed

**Comment by Pilow974** [comment]
> @Pilow974 it is possible to have the same translated in EN?

@lpinsivy  It's done, ready for review

**Comment by quanghungb** [comment]
Hello 
Does it mean that the Centreon source is stopped ? No running monitoring otherwise data will be lost ? (during the export from source et import to dest database) 

Regards.

**Comment by Pilow974** [comment]
Yes, source centreon is stopped during the migration.
I would advise to test-proof the migration first (let source database up) and check how long and if successfull the migration is.
Once migrated successfully, you know the aprox. time it will long. You can now choose the best timeframe to migrate your Centreon to 18.10.

Your pollers should do retention as soon as source Centreon is stopped. When migrated Centreon is up they will deliver the updates to new database. Central poller will lose updates (move hosts to distant poller before migrating ?)

File: doc/fr/migration/upgradetoCentreon18.10.rst (hunk 1 above)
Comment (Pilow974) [reply doc/fr/migration/upgradetoCentreon18.10.rst]: changes are made, can you update the review @lpinsivy

**Pull request merged**